src/mudlet-mapper.xml

```diff
@@ -3549,6 +4231,112 @@ end</script>
                 <packageName></packageName>
                 <regex>^room marks$</regex>
             </Alias>
+            <Alias isActive="yes" isFolder="no">
+                <name>Toggle map sizes</name>
+                <script>local size = matches[2]
+
+if size == &quot;biggest&quot; then
+  mapper = Geyser.Mapper:new({
+    name = &quot;mapper&quot;,
+    x = 0, y = 0,
+    width = &quot;100%&quot;, height = &quot;97%&quot;
+  })
+  mmp.echo(&quot;Map size set to pretty damn big.&quot;)
+
+elseif size == &quot;small&quot; then
+  mapper = Geyser.Mapper:new({
+    x = &quot;70%&quot;, y = 0,
+    width = &quot;28%&quot;, height = &quot;50%&quot;
+  })
+  mmp.echo(&quot;Map size set to comfortable.&quot;)
+
+elseif size == &quot;big&quot; then
+  local window_width, window_height = getMainWindowSize()
+  local used_width = getMainConsoleWidth()
+
+  local available_space = window_width - used_width
+
+  if matches[3] then available_space = available_space - tonumber(matches[3]) end
+
+  mapper = Geyser.Mapper:new({
+    x = available_space*-1, y = 0,
+    width = available_space, height = &quot;100%&quot;
+  })
+
+  mmp.echo(&quot;Set the map size to big - it'll cover all of the space on the right that game text isn't using. You'll want to call this alias again if you resize the window to update.&quot;)
+end</script>
+                <command></command>
+                <packageName></packageName>
+                <regex>^(small|big|biggest)map(?: (\d+))?$</regex>
+            </Alias>
+            <Alias isActive="yes" isFolder="no">
+                <name>^make maps$</name>
+                <script>local c = 0
+for _, id in pairs(getAreaTable()) do
+  tempTimer(c*.5, [[exportAreaImage(]]..id..[[)]])
+  c = c + 1
+end
+mmp.echo(&quot;Done!&quot;)</script>
+                <command></command>
+                <packageName></packageName>
+                <regex>^make maps$</regex>
+            </Alias>
+            <Alias isActive="yes" isFolder="no">
+                <name>Set debug mode</name>
+                <script>if matches[2] == &quot;on&quot; then
+  mmp.debug = true
+else
+  mmp.debug = false
+end
+
+mmp.echo(&quot;Debug &amp; performance telemetry &quot;..(mmp.debug and &quot;enabled&quot; or &quot;disabled&quot;)..&quot;.&quot;)</script>
+                <command></command>
+                <packageName></packageName>
+                <regex>^(?:mdg|mdebug) (on|off)$</regex>
+            </Alias>
+            <Alias isActive="yes" isFolder="no">
+                <name>measure getPath()</name>
+                <script>mmp = mmp or {}
+
+mmp.getPathPerf = mmp.getPathPerf or createStopWatch()
+startStopWatch(mmp.getPathPerf)
+
+local from, to = tonumber(matches[2]), tonumber(matches[3])
+
+getPath(from, to)
```

Comment:
Don't forget you now have the global `speedWalkWeight` table as well now with the weights of each step and the `getPath()` function also has a return value of the total weight of the route. :smiley: